cookbooks/bach_ambari/attributes/default.rb

```diff
@@ -0,0 +1,45 @@
+node.force_default['ambari']['install_java'] = false
+node.force_default['ambari']['java_home'] = '/usr/lib/jvm/java-8-oracle-amd64/'
```

Comment:
This should come from [BCPC-Hadoop](https://github.com/bloomberg/chef-bach/blob/39fc7bbb30af803482308b08cc25955671bd5926/cookbooks/bcpc-hadoop/attributes/default.rb#L173)

Reply:
Please see [bcpc-hadoop's default attributes file](https://github.com/bloomberg/chef-bach/blob/998a982e77aecad7949a6918145e9c2065d55629/cookbooks/bcpc-hadoop/attributes/default.rb#L172-L173)